controllers/controllers.js

```diff
@@ -68,3 +69,14 @@ exports.getCommentsById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  insertComment(req.body, article_id)
+    .then(([comment]) => {
+      res.status(201).send({ comment:comment.body });
```

Comment:
Looks like here, as long as you amend the test first, is where you can simply send back the whole comment

Reply:
- [ X ]